apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/r4/providers/R4ExplanationOfBenefitResourceProvider.java

```diff
@@ -347,12 +348,17 @@ public Bundle findByPatient(
           paging, eobs, loadedFilterManager.getTransactionTime());
     }
 
+    // See if we have claims data for the beneficiary.
+    BitSet bitSet = QueryUtils.hasClaimsData(entityManager, beneficiaryId);
+    LOGGER.info(
+        String.format("BitSet for V2 claims, bene_id %d: %s", beneficiaryId, bitSet.toString()));
```

Comment:
Should be a debug or somesuch; Doubt we want this clogging the logs for every request.

Reply:
I made them both debug message(s), but reluctantly. To me this had value in that it logged the bene_id AND the mask (i.e., {0, 1, 3, 5, 7} ) that would be applied to the EOB processing. That mask provided forensic info that might be useful in tracking down a problem.

Reply:
burling is already talking about a follow-up; basically perform all the claim types to be used in parallel.